torchtext/csrc/vocab.cpp

```diff
@@ -98,6 +98,160 @@ std::vector<int64_t> Vocab::lookup_indices(const StringList &tokens) {
 c10::Dict<std::string, int64_t> Vocab::get_stoi() const { return stoi_; }
 StringList Vocab::get_itos() const { return itos_; }
 
+inline int64_t divup(int64_t x, int64_t y) { return (x + y - 1) / y; }
```

Comment:
There seems to be significant codeshare between this and vocab. Can we just share these functions with the other file?

Reply:
Added some of the duplicated code into `torchtext::impl` namespace. Can you take one more look and lmk what you think?